tests/e2e/src/create_test_data/createTestDataManage.js

```diff
@@ -18,72 +18,71 @@ const transactionInfo = {
   transactionIndex: "",
   accountAddress: "",
 };
-
-const savejson = async (filePath, data) => {
-  const dataStr = JSON.stringify(data, null, 4);
-  if (dataStr) {
+const createTestDataMange = {
```

Comment:
Maybe exporting these methods separately is better? I can't get this.

Reply:
This is more conducive to object encapsulation. In addition, the code is also more concise, and the place where this method is called is more clear about the method is belongs to which object.

Reply:
> Maybe exporting these methods separately is better? I can't get this.

This is more conducive to object encapsulation. In addition, the code is also more concise, and the place where this method is called is more clear about the method is belongs to which object.